mk/spksrc.install.mk

```diff
@@ -5,25 +5,30 @@
 # Targets are executed in the following order:
 #  install_msg_target
 #  $(PRE_INSTALL_PLIST)
-#  pre_install_target              (override with PRE_INSTALL_TARGET)
-#  install_destdir_target          (default override with INSTALL_TARGET)
-#  install_target                  (alternative for INSTALL_TARGET)
-#  post_install_target             (override with POST_INSTALL_TARGET)
+#  pre_install_target               (override with PRE_INSTALL_TARGET)
+#  install_target                   (default override with INSTALL_TARGET)
+#  post_install_target              (override with POST_INSTALL_TARGET)
 #  $(INSTALL_PLIST)
 #  install_correct_lib_files
 # Variables:
-#  INSTALL_PREFIX          Target directory where the software will be run.
-#  INSTALL_DIR             Where to install files. INSTALL_PREFIX will be added.
-#  STAGING_INSTALL_PREFIX  Where to install files, in extenso.
+#  INSTALL_PREFIX                   Target directory where the software will be run.
+#  INSTALL_DIR                      Where to install files. INSTALL_PREFIX will be added.
+#  STAGING_INSTALL_PREFIX           Where to install files, in extenso.
+#  INSTALL_MAKE_OPTIONS             Parameters to add to make command
+#                                   (default: install DESTDIR=$(INSTALL_DIR) prefix=$(INSTALL_PREFIX))
```

Comment:
That's exactly what I was looking for... Once merged can you also update the wiki page?